tracks/javascript/exercises/pascals-triangle/mentoring.md

```diff
@@ -1,47 +1,44 @@
 ### Reasonable solutions
 
 ```javascript
-function buildRow(result, element, index, array) {
-  return result.concat([element + (array[index + 1] || 0)])
+const buildRow = (result, element, index, array) => {
```

Comment:
```suggestion
const buildRow = (result, element, index, array) => {
```

This is _as good_ as using a `function declaration`. It's a good idea to mention below the solution that any way of writing a function is fine.

Reply:
Indeed, but here, the aim was to have a consistent style on this site where we only used arrow function in exercises templates...

And, is this the exercise to mention this point? This exercise comes really late and I guess that we (JS mentors) have already spoken about standard / arrow functions, don't you think so?

Reply:
I just mention it always, because some mentors start mentoring late exercises.